palette/src/convert.rs

```diff
@@ -201,15 +199,10 @@ use encoding::Linear;
 ///     let color = LinSrgba::new(0.5, 0.0, 1.0, 0.3);
 ///     let css_color = CssRgb::from(color);
 ///
-///     assert_eq!(
-///         css_color,
-///         CssRgb {
-///             red: 188,
-///             green: 0,
-///             blue: 255,
-///             alpha: 0.3,
-///         }
-///     );
+///     assert_eq!(css_color.red, 188);
+///     assert_eq!(css_color.green, 0);
+///     assert_eq!(css_color.blue, 255);
+///     assert_relative_eq!(css_color.alpha, 0.3);
```

Comment:
I think this one is safe to make an exception for. It's not going to alter the alpha value in any way.